package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dmptool/types",
-  "version": "1.0.4",
+  "version": "1.0.7",
```

Comment:
looks like I hadn't pushed the version change last time. NPM currently has 1.0.6.